index.js

```diff
@@ -0,0 +1,2 @@
+//use strict
```

Comment:
The strict mode you apply here really makes sense

Reply:
Yea bro, thanks for your review